mailet/base/src/test/java/org/apache/mailet/base/test/MimeMessageBuilder.java

```diff
@@ -305,22 +325,21 @@ public MimeMessage build() throws MessagingException {
         Preconditions.checkState(!(text.isPresent() && content.isPresent()), "Can not get at the same time a text and a content");
         MimeMessage mimeMessage = new MimeMessage(Session.getInstance(new Properties()));
         if (text.isPresent()) {
-            BodyPart bodyPart = new MimeBodyPart();
-            bodyPart.setText(text.get());
-            mimeMessage.setContent(bodyPart, "text/plain");
+            mimeMessage.setContent(text.get(), "text/plain");
```

Comment:
I'm not sure this fix is fine. When you get a mail, you don't get it with a textBody, you get a BinaryBody with an InputStream you need to parse. So I think we are trying to mimic this behaviour.
Can you check please?